commands.js

```diff
@@ -1,13 +1,28 @@
 'use strict';
 const fs = require('fs');
 const chalk = require('chalk');
+const pathFile = 'input.json';
+
+function ifExist(path) {
+  try {
+    if (fs.existsSync(path)) {
+      return;
+    } else {
+      fs.writeFileSync(path, '[]');
+    }
+  } catch (error) {
+    console.log(error);
+  }
+}
 
 function readFile(path) {
+  ifExist(path);
```

Comment:
There are repetitions, which we will eliminate in the future. For now, it's ok :)